enhancements/windows-containers/container-runtime-containerd.md

```diff
@@ -118,13 +122,10 @@ Steps to install containerd as service:
 ## Network changes
 Current CNI/IPAM will be used for containerd and no changes will be made to HNS-Network and HNS-Endpoint creation
 steps. The config file which will be used by containerd will point to the same CNI/IPAM executables.
+As the current CNI V1 APIs doesn't support containerd in Windows, V2 API support has to be implemented in CNI.
```

Comment:
`V2 API support has to be implemented in CNI.`
How much work is required here?

Reply:
It has already been implemented and we are using it. Can we state it that way here?

Reply:
@selansen can we say it has already been implemented, saying 'has to be implemented' may be confusing.

Reply:
and perhaps link the PR where y'all implemented it